Add `osu!mania 7K World Cup 2024: Semifinals Recap` news post (#10982)
ppy/osu-wiki: news/2024/2024-02-16-osumania-7k-world-cup-2024-semifinals-recap.md

---
layout: post
title: "osu!mania 7K World Cup 2024: Semifinals Recap"
date: 2024-02-16 20:00:00 +0000
---

Catch up with the latest MWC7K happenings, and join us in preparations for the upcoming Finals weekend!

![](/wiki/Tournaments/MWC/2024_7K/img/mwc7k2024-banner.jpg)

> The Asian countries have once again shown complete and utter dominance when it comes to 7K mania, once again not allowing any other region to make it past the Semifinals (unlike last year) — and oh boy do next weeks' match-ups look exciting!

Reviewer: ```suggestion
The Asian countries have once again shown complete and utter dominance when it comes to 7K osu!mania, once again not allowing any other region to make it past the Semifinals (unlike last year) — and oh boy do next weeks' match-ups look exciting!
```

However, before looking forward we shall look back once more at the previous week, and everything brought forward by it!

## Mappool showcase

[**Download the pool here! (119 MB)**](https://osu.ppy.sh/beatmaps/packs/P236)

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=v2059913783&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

Semifinals not only turned up the difficulty one or two notches, but it was also the first round where the tournament's first [osu! originals](/wiki/osu!_originals) were showcased! Not only that, but we had [Akiri](https://osu.ppy.sh/beatmaps/artists/401)'s big Featured Artist debut as well.

> Feel free to check out both the original tracks and mapsets by clicking the banners below. Alternatively, we'd encourage to give the audio previews a shot!

Reviewer: ```suggestion
Feel free to check out both the original tracks and mapsets by clicking the banners below. For a quick taste, you can also listen to samples of them right here with the audio previews!
```

[![](/wiki/shared/news/2024-02-16-osumania-7k-world-cup-2024-semifinals-recap/Kagaribi.jpg)](https://osu.ppy.sh/beatmapsets/2130665#mania/4481018)

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/artists/365/Songs/Junk_-_Kagaribi.mp3">
    </audio>
</div>

[![](/wiki/shared/news/2024-02-16-osumania-7k-world-cup-2024-semifinals-recap/Symphony-Of-The-Lost-Souls.jpg)](https://osu.ppy.sh/beatmapsets/2130621#mania/4480922)

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/artists/401/The%20Lost%20Symphonies/Akiri%20-%20Symphony%20Of%20The%20Lost%20Souls.mp3">
    </audio>
</div>

[![](/wiki/shared/news/2024-02-16-osumania-7k-world-cup-2024-semifinals-recap/Voyage.jpg)](https://osu.ppy.sh/beatmapsets/2130624#mania/4480927)

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/artists/392/Songs/Sydosys_-_Voyage.mp3">
    </audio>
</div>

## Semifinals results

> As the final rounds draw nigh, all the teams are starting to show their true colours as we reach endgame levels of difficulty. As such, those who were able to keep up with the sharp increase in difficulty had an easier time this round. Because of that, some teams that initially did quite well in earlier rounds—despite giving their all — weren't able to make a dent on some of the behemoths they were up against.

Reviewer: ```suggestion
As the final rounds draw nigh, all the teams are starting to show their true colours as we reach endgame levels of difficulty. As such, those who were able to keep up with the increased challenge had an easier time this round. Because of that, some teams that initially did quite well in earlier rounds — despite giving their all — weren't able to make a dent on some of the behemoths they were up against.
```

| Team A |  |  | Team B |
| --: | :-: | :-: | :-- |
| **Australia** ::{ flag=AU }:: | **6** | 1 | ::{ flag=US }:: United States |
| **Japan** ::{ flag=JP }:: | **6** | 1 | ::{ flag=RU }:: Russian Federation |
| **Thailand** ::{ flag=TH }:: | **6** | 0 | ::{ flag=PL }:: Poland |
| China ::{ flag=CN }:: | 4 | **6** | ::{ flag=MY }:: **Malaysia** |
| **Canada** ::{ flag=CA }:: | **6** | 1 | ::{ flag=BR }:: Brazil |
| **Japan** ::{ flag=JP }:: | **6** | 4 | ::{ flag=AU }:: Australia |
| **South Korea** ::{ flag=KR }:: | **6** | 0 | ::{ flag=PH }:: Philippines |
| **Thailand** ::{ flag=TH }:: | **6** | 2 | ::{ flag=CA }:: Canada |

## Match highlights

> ### China ::{ flag=CN }:: vs. ::{ flag=MY }:: Malaysia

Reviewer: wrong way round

```suggestion
###  ::{ flag=CN }:: China vs. ::{ flag=MY }:: Malaysia
```

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=v2058868470&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

China vs. Malaysia was a long-awaited clash between two 7K titans. These two teams have been consistent contenders for the MWC7K podium. China, the 2016 champions — came into this match having narrowly escaped with a tiebreaker win in the Quarterfinals. The Malaysian duo, on the other hand, will finally be tested against a team of their calibre.

> These two countries are known as the 'noodle countries' due to their VSRG history. As such, this match was initially seen as a 'who will choke on their rice first' affair. As expected, Malaysia opened with LN3, [GHOST](https://osu.ppy.sh/beatmapsets/2130609#mania/4480895), but it was China who drew first blood. The noodle-slurping action continued as China started with LN2, [Vulnerability](https://osu.ppy.sh/beatmapsets/2130611#mania/4480897), which happened to be their protected pick for this match. However, Malaysia were able to snatch this table away from them. Both teams then traded break points on the next two charts to level the match at 2–2.

Reviewer: ```suggestion
These two are known as the "noodle countries" due to their VSRG history. As such, this match was initially seen as a "who will choke on their rice first" affair. As expected, Malaysia opened with LN3, [Hoshimachi Suisei - GHOST](https://osu.ppy.sh/beatmapsets/2130609#mania/4480895), but it was China who drew first blood. The noodle-slurping action continued as China started with LN2, [Vulnerability](Mili - https://osu.ppy.sh/beatmapsets/2130611#mania/4480897), which happened to be their protected pick for this match. However, Malaysia was able to snatch this table away from them. Both teams then traded break points on the next two charts to level the match at 2–2.
```

Author: ```suggestion
These two countries are known as the 'noodle countries' due to their VSRG history. As such, this match was initially seen as a 'who will choke on their rice first' affair. As expected, Malaysia opened with LN3, [GHOST](https://osu.ppy.sh/beatmapsets/2130609#mania/4480895), but it was China who drew first blood. The noodle-slurping action continued as China started with LN2, [Mili - Vulnerability](https://osu.ppy.sh/beatmapsets/2130611#mania/4480897), which happened to be their protected pick for this match. However, Malaysia were able to snatch this table away from them. Both teams then traded break points on the next two charts to level the match at 2–2.
```
i think you put the text in the wrong place

> The match continued with this tug of war until the LN and Hybrid picks ran out. At the end of LN4, [Revolt](https://osu.ppy.sh/beatmapsets/2130612#mania/4480900), China led 4–3, and it was their turn to pick. They chose RC6, [Kagaribi](https://osu.ppy.sh/beatmapsets/2130665#mania/4481018), which did not work out well for them as Malaysia managed to level this match. The next two picks came from the rice side of the pool. RC4, [Voyage](https://osu.ppy.sh/beatmapsets/2130624#mania/4480927), once again ended in Malaysia's favour.

Reviewer: ```suggestion
The match continued with this tug of war until the LN and Hybrid picks ran out. At the end of LN4, [Getty - Revolt](https://osu.ppy.sh/beatmapsets/2130612#mania/4480900), China led 4–3, and it was their turn to pick. They chose RC6, [Junk - Kagaribi](https://osu.ppy.sh/beatmapsets/2130665#mania/4481018), which did not work out well for them as Malaysia managed to level the match. The next two picks came from the rice side of the pool. RC4, [Sydosys - Voyage](https://osu.ppy.sh/beatmapsets/2130624#mania/4480927), once again ending in Malaysia's favour.
```

> China's next selection had to go well for them to force a tiebreaker. They chose RC3, [tinnitus](https://osu.ppy.sh/beatmapsets/2130608#mania/4480894), hoping to extend the match by one more map. In the end, it was Malaysia who closed out this match 4–6. ::{ flag=MY }:: [cheewee10](https://osu.ppy.sh/users/4477497) lead the way with 10 MVP performances in 10 charts played, which is a particularly impressive showing in a match with two high-calibre teams.

Reviewer: ```suggestion
China's next selection had to go well for them to force a tiebreaker. They chose RC3, [SP-# - tinnitus](https://osu.ppy.sh/beatmapsets/2130608#mania/4480894), hoping to extend the match by one more map. In the end, it was Malaysia who closed out the match 4–6. ::{ flag=MY }:: [cheewee10](https://osu.ppy.sh/users/4477497) lead the way with 10 MVP performances in 10 charts played, which is a particularly impressive showing in a match with two high-calibre teams.
```

> Malaysia advanced to the Winner's Bracket Final where they will face perennial contenders South Korea for a place in the Grand Finals. China, on the other hand, will be in for an intense campaign in the Loser's Bracket.

Reviewer: ```suggestion
Malaysia advanced to the Winners' Bracket Final where they will face perennial contenders South Korea for a place in the Grand Finals. China, on the other hand, will be in for an intense campaign in the losers' bracket.
```

> ### Japan ::{ flag=JP }:: vs. ::{ flag=AU }:: Australia

Reviewer: ```suggestion
### ::{ flag=JP }:: Japan vs. ::{ flag=AU }:: Australia
```

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=v2059665757&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

Japan vs. Australia was yet another match many people were looking for. There were several reasons for this, such as their similarly seeded placement or the amazing performances they've already shown throughout the tournament. Suffice to say, the anticipation was overwhelming — and it didn't disappoint!

> The teams started on a good foot by choosing their bans, protects, and first pick wisely. Australia had an edge on long note picks, whereas Japan has been consistently demolishing rice maps on all rounds. That's why on their first pick, they opened up with [Sydosys - Voyage](https://osu.ppy.sh/beatmapsets/2130624#mania/4480927). And ::{ flag=JP }:: [rainlily](https://osu.ppy.sh/users/485007)'s score should've been indicative of Japan's rice prowess right off the bat.

Reviewer: ```suggestion
The teams started on good feet by choosing their bans, protects, and first picks wisely. Australia had an edge on long note picks, whereas Japan has been consistently demolishing rice maps on all rounds. That's why on their first pick, they opened up with [Sydosys - Voyage](https://osu.ppy.sh/beatmapsets/2130624#mania/4480927), and ::{ flag=JP }:: [rainlily](https://osu.ppy.sh/users/485007)'s score should've been indicative of Japan's rice prowess right off the bat.
```

> The match went on with Australia picking [ARForest - The Last Page](https://osu.ppy.sh/beatmapsets/2130622#mania/4480923), securing their protect and answering to Japan's protect, and ::{ flag=AU }:: [ruka](https://osu.ppy.sh/users/6117525) showing off Australia's strong suit in long note maps.

Reviewer: ```suggestion
The match went on with Australia picking [ARForest - The Last Page](https://osu.ppy.sh/beatmapsets/2130622#mania/4480923), securing their protect and answering to Japan's protect, where ::{ flag=AU }:: [ruka](https://osu.ppy.sh/users/6117525) could show off Australia's strong suit in long note maps.
```

> The rice vs. long note back and forth went on, with Japan picking [Junk - Kagaribi](https://osu.ppy.sh/beatmapsets/2130665#mania/4481018), Australia answering with [Getty - Revolt](https://osu.ppy.sh/beatmapsets/2130612#mania/4480900), then [RykSeb - RETINAE](https://osu.ppy.sh/beatmapsets/2130601#mania/4480882). Australia gambled their chances on [tn-shi - Synthesis](https://osu.ppy.sh/beatmapsets/2130789#mania/4481282) and managed to survive a potential break point on what ended up being the closest pick yet!

Reviewer: ```suggestion
The rice vs. long note back-and-forth went on, with Japan picking [Junk - Kagaribi](https://osu.ppy.sh/beatmapsets/2130665#mania/4481018), Australia answering with [Getty - Revolt](https://osu.ppy.sh/beatmapsets/2130612#mania/4480900), then [RykSeb - RETINAE](https://osu.ppy.sh/beatmapsets/2130601#mania/4480882). Australia gambled their chances on [tn-shi - Synthesis](https://osu.ppy.sh/beatmapsets/2130789#mania/4481282) and managed to survive a potential break point on what ended up being the closest pick yet!
```

Unfortunately for Australia, despite making a huge effort by keeping up with Japan, when Japan managed to get a match point at 5–4, Australia wasn't able to break through their pick, [Frums - 24eeev0-$](https://osu.ppy.sh/beatmapsets/2130620#mania/4480921), and ended up losing the break point.

> Despite how the match ended, it is safe to assume nobody was sure of who was going to be the winner given both teams' phenomenal performance. A nerve-wracking match full and through.

Reviewer: ```suggestion
Despite how the match ended, it is safe to assume nobody was sure of who was going to be the winner given both teams' phenomenal performance. A nerve-wracking match through and through.
```

---

> The tournament is slowly coming to a close, with 2 more weeks left ahead of us. It would be a real shame if you made it this far without checking out any of the matches live, so be sure to visit the [osulive Twitch channel](https://www.twitch.tv/osulive) this weekend for some high intensity mania action! If you are interested in more details regarding the tournament, such as the Finals pool or the full bracket — the [wiki article](/wiki/Tournaments/MWC/2024_7K) has everything you need!

Reviewer: ```suggestion
The tournament is slowly coming to a close, with 2 more weeks left ahead of us. It would be a real shame if you made it this far without checking out any of the matches live, so be sure to visit the [osulive Twitch channel](https://www.twitch.tv/osulive) this weekend for some high-intensity osu!mania action!

If you are interested in more details regarding the tournament, such as the Finals pool or the full bracket — the [wiki article](/wiki/Tournaments/MWC/2024_7K) has everything you need! (It also has the match schedules, which are updated regularly, so you probably want to check it out anyway to know when things go live!)
```
